tools/msvs/msi/product.wxs

```diff
@@ -2,11 +2,11 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
      xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
 
-  <?define ProductName = "Node.js" ?>
-  <?define ProductDescription = "Node.js" ?>
-  <?define ProductAuthor = "Joyent, Inc. and other Node contributors" ?>
+  <?define ProductName = "io.js" ?>
+  <?define ProductDescription = "io.js" ?>
+  <?define ProductAuthor = "io.js contributors" ?>
 
-  <?define RegistryKeyPath = "SOFTWARE\Node.js" ?>
+  <?define RegistryKeyPath = "SOFTWARE\io.js" ?>
```

Comment:
@piscisaureus any idea what the implications of changing the registry key path might be?